documentation/design-docs/ipc-protocol.md

```diff
@@ -376,10 +376,11 @@ See: [Profiler Commands](#Profiler-Commands)
 ```c++
 enum class ProcessCommandId : uint8_t
 {
-    ProcessInfo        = 0x00,
-    ResumeRuntime      = 0x01,
-    ProcessEnvironment = 0x02,
-    ProcessInfo2       = 0x04,
+    ProcessInfo             = 0x00,
+    ResumeRuntime           = 0x01,
+    ProcessEnvironment      = 0x02,
+    ProcessInfo2            = 0x04,
+    GetAppContextProperties = 0x07
```

Comment:
This should be ApplyStartupHook now, right? And is there a reason you skipped numbers? These aren't flags, you can pick the next available

Reply:
> This should be ApplyStartupHook now, right?

Correct, I wonder if I forgot to commit the rename. I'll get it fixed today.

> And is there a reason you skipped numbers? These aren't flags, you can pick the next available

The runtime repo already has several commands that have not been added to diagnostics repo yet. I chose the next available in runtime and reflected that here.

Reply:
See https://github.com/dotnet/runtime/blob/bace87457f8545b7469cb427f591039dd05fb78a/src/native/eventpipe/ds-types.h#L79 for the existing commands that haven't been reflected here yet.